node/node.go

```diff
@@ -27,7 +30,9 @@ type Node struct {
 	conf config.NodeConfig
 	P2P  *p2p.Client
 
-	Mempool mempool.Mempool
+	Mempool       mempool.Mempool
+	mempoolIDs    *mempoolIDs
+	incommingTxCh chan *p2p.Tx
```

Comment:
So these 3 fields basically acts as the mempool reactor. I'm wondering if we should capture them in a struct together too 🤔